### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Select Source-Target_Schema Compare_ColorContrast: When applied High contrast theme and navigated to grid content, focus is not visible on each cell.

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6852]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAS References: MAS1.4.3

**Repro Steps:**
Launch Azure Data Studio application.
Connect to server with windows credentials.
Navigate to Extensions from left menu.
Search & install Schema Compare extension.
Navigate to Connections from left menu.
Expand the server connected in step 2.
Right click an on prem database and select Schema compare from Context menu.
Navigate till Select Source or Select Target & hit enter.
9.Navigate till Compare button & hit enter.
Apply High Contrast theme.
Navigate till grid content using tab key.

**Actual**:
When applied High contrast theme and navigated to grid content, focus is not visible on each cell.

**Expected:**
When applied High contrast theme and navigated to grid content, focus should be clearly visible on each cell.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Low vision users will face difficulty to trace current focus while navigating to each cell of a grid.

**MAS Reference:**
MAS 1.4.3-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment for Reference:**
[A11y_AzureDataStudio.Schema.Dacpac.Backup._Select.Source-Target_Schema.Compare_ColorContrast.pptx](https://github.com/microsoft/azuredatastudio/files/3525281/A11y_AzureDataStudio.Schema.Dacpac.Backup._Select.Source-Target_Schema.Compare_ColorContrast.pptx)

**Comment by udeeshagautam** [comment]
@Andrew gave some suggestions... will work accordingly

**Comment by udeeshagautam** [comment]
Discussed with @angdesign for some options... will followup/work accordingly.

**Comment by fsteffi** [comment]
Checked and verified in below version the bug is no longer reproduced and hence closing the bug.

Version Details:Version: 1.12.0-insider
Commit: 8e1a2248e47925eb825b0ce037b96b6c87798a8a
Date: 2019-10-04T07:52:03.973Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
[#6852A11y_AzureDataStudio(Schema,Dacpac,Backup)_Select Source-Target_Schema Compare_ColorContrast.pptx](https://github.com/microsoft/azuredatastudio/files/3715789/6852A11y_AzureDataStudio.Schema.Dacpac.Backup._Select.Source-Target_Schema.Compare_ColorContrast.pptx)
